src/app/photos/photos.component.html

```diff
@@ -0,0 +1,12 @@
+<li>
+    <h2><a target="_blank" rel="noopener" href="https://blog.angular.io/">Angular blog</a></h2>
+</li>
+<button (click)="getPhotos()">Get Photos</button>
+<div class="photo">
+    <h1>{{photo.image}}</h1>
+    <p>{{photo.link}}</p>
+    <p>{{photo.name}}</p>
+</div>
+</ul>
+
+<router-outlet></router-outlet>
```

Comment:
Considering removing `<router-outlet></router-outlet>` here since we do not intend to nest a routing from photos page.